src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -130,6 +130,44 @@ Copyright (c) .NET Foundation. All rights reserved.
     <IncludeProjectsNotInAssetsFileInDepsFile Condition="'$(IncludeProjectsNotInAssetsFileInDepsFile)' == ''">true</IncludeProjectsNotInAssetsFileInDepsFile>
   </PropertyGroup>
 
+  <!--
+    BinaryFormatter killbitting
```

Comment:
```suggestion
    BinaryFormatter Disabling
```

Reply:
Good catch on this, @JeremyKuhne. @GrabYourPitchforks, I've updated the PR title and description to use "disable" verbiage.